src/Utils/FileSystem.php

```diff
@@ -88,6 +88,17 @@ public static function delete(string $path): void
 	}
 
 
+	/**
+	 * Moves a file or directory.
+	 * @throws Nette\IOException
+	 */
+	public static function move(string $source, string $dest, bool $overwrite = true): void
+	{
+		static::copy($source, $dest, $overwrite);
```

Comment:
Vhodnější je použít rename(), protože copy() je mnohem pomalejší.

Reply:
Better is try rename and then if it fail (eg. two filesystems) try copy.